accounts/usbwallet/hub.go

```diff
@@ -151,7 +151,7 @@ func (hub *Hub) refreshWallets() {
 	hub.stateLock.Lock()
 
 	wallets := make([]accounts.Wallet, 0, len(devices))
-	events := []accounts.WalletEvent{}
+	var events []accounts.WalletEvent
```

Comment:
This has the same issue that I pinged you about the previous segment :P

Reply:
Done. Apologies I misunderstood that the transformation in `keystore.go` should be applied here.